silver/models/payment_methods.py

```diff
@@ -55,6 +55,9 @@ def as_list(cls):
     verified = models.BooleanField(default=False)
     canceled = models.BooleanField(default=False)
 
+    valid_until = models.DateTimeField(null=True, blank=True)
+    display_info = models.CharField(max_length=256, null=True, blank=True)
```

Comment:
How about we use https://github.com/silverapp/silver/blob/master/silver/models/payment_methods.py#L140 which can get it's data from the `data` field. In this way we store data in one place and have it in a serializable form.